conans/test/utils/tools.py

```diff
@@ -462,205 +452,8 @@ def package_revision_time(self, pref):
-        tmp_dir = self.gimme_tmp()
-        return create_local_svn_checkout(files, self.repo_url, rel_project_path=rel_project_path,
-                                         commit_msg=commit_msg, delete_checkout=delete_checkout,
-                                         folder=tmp_dir)
-
-    def run(self, *args, **kwargs):
-        tmp_folder = tempfile.mkdtemp(suffix='_conans')
-        try:
-            self._tmp_folder = os.path.join(tmp_folder, 'path with spaces'
-                                            if self.path_with_spaces else 'pathwithoutspaces')
-            os.makedirs(self._tmp_folder)
-            self.repo_url = self._create_local_svn_repo()
-            super(SVNLocalRepoTestCase, self).run(*args, **kwargs)
-        finally:
-            shutil.rmtree(tmp_folder, ignore_errors=False, onerror=try_remove_readonly)
-
-
-class LocalDBMock(object):
-
-    def __init__(self, user=None, access_token=None, refresh_token=None):
-        self.user = user
-        self.access_token = access_token
-        self.refresh_token = refresh_token
-
-    def get_login(self, _):
-        return self.user, self.access_token, self.refresh_token
-
-    def get_username(self, _):
-        return self.user
-
-    def store(self, user, access_token, refresh_token, _):
-        self.user = user
-        self.access_token = access_token
-        self.refresh_token = refresh_token
-
-
-class MockedUserIO(UserIO):
-    """
-    Mock for testing. If get_username or get_password is requested will raise
-    an exception except we have a value to return.
-    """
-
-    def __init__(self, logins, ins=sys.stdin, out=None):
-        """
-        logins is a dict of {remote: list(user, password)}
-        will return sequentially
-        """
-        assert isinstance(logins, dict)
-        self.logins = logins
-        self.login_index = Counter()
-        UserIO.__init__(self, ins, out)
-
-    def get_username(self, remote_name):
-        username_env = self._get_env_username(remote_name)
-        if username_env:
-            return username_env
-
-        self._raise_if_non_interactive()
-        sub_dict = self.logins[remote_name]
-        index = self.login_index[remote_name]
-        if len(sub_dict) - 1 < index:
-            raise Exception("Bad user/password in testing framework, "
-                            "provide more tuples or input the right ones")
-        return sub_dict[index][0]
-
-    def get_password(self, remote_name):
-        """Overridable for testing purpose"""
-        password_env = self._get_env_password(remote_name)
-        if password_env:
-            return password_env
-
-        self._raise_if_non_interactive()
-        sub_dict = self.logins[remote_name]
-        index = self.login_index[remote_name]
-        tmp = sub_dict[index][1]
-        self.login_index.update([remote_name])
-        return tmp
-
-
 def _copy_cache_folder(target_folder):
+    _copy_cache_folder.master = dict()  # temp_folder(create_dir=False)
```

Comment:
Revert this change, this will clean and remove existing entries in the dictionary every time and will use a new folder each time.

Reply:
Yes, fixed, this didn't belong to this PR, it was a half-baked thing.

Reply:
Yes, absolutely no code changes, just moving things around.